### ISSUE TITLE
androidx.fragment:fragment:1.3.0-beta01@aar@aar as dependency in Netaxept 2.1.3

### ISSUE THREAD

**Issue by mkovalyk** [Nets-mobile-acceptance/Netaxept-Android-SDK#24]
Hi there. I have just updated the library to **2.1.3** and turns out it uses beta version of androidx.fragment library:  **androidx.fragment:fragment:1.3.0-beta01@aar**
Of course, there is a way to exclude it from the dependency, but I wonder if it won't break anything in the library itself. 
Do you really need this dependency? Are there any plans not to use *beta* version? 

Reason for ticket:  Crash when opening the opening ours fragments:
```
Process: PID: 6087
    java.lang.IllegalStateException: Fragment xxx{4ce981b} (751bad62-9447-47c8-9419-852ec1c7c122)} has not been attached yet.
        at androidx.fragment.app.Fragment.getChildFragmentManager(Fragment.java:1073)
        at xxx.containers.ContainerFragment.onHiddenChanged(ContainerFragment.kt:57)
        at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:374)
        at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2177)
        at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2088)
        at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1990)
        at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:524)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7711)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:516)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
```

**Comment by Miss-PiA** [comment]
Hi There,

Thank you for reaching out.

For now, the dependency could be excluded and this will not affect the SDK functionality. 

We are looking into the dependencies and will move to the latest stable version in the next release.

Best Regards
SDK Team

**Comment by Miss-PiA** [comment]
Hello @mkovalyk ,

We have resolved this in version 2.2.0. Please check the same and let us know if you face any issue.

Best Regards.
